src/pages/workspaces/workspace/Dashboard/WorkspaceInformation.test.ts

```diff
@@ -99,4 +101,42 @@ describe('WorkspaceInformation', () => {
     // Accessibility
     expect(await axe(container)).toHaveNoViolations();
   });
+
+  it.each([
+    { policies: [] },
+    {
+      policies: [
+        {
+          namespace: 'terra',
+          name: 'group-constraint',
+          additionalData: [{ group: 'foo' }],
+        },
+        {
+          namespace: 'terra',
+          name: 'group-constraint',
+          additionalData: [{ group: 'foo' }, { group: 'bar' }],
```

Comment:
I don't know if a group constraint policy can actually have multiple groups listed in `additionalData`.

Reply:
Can you ask folks on your team to see if we should expect multiple "group-constraint" policies or just a single one with multiple groups?

Reply:
It can be both. There can be multiple group-constraint policies, and each of those policies can have multiple groups underneath it.

Reply:
Ah, thanks @aherbst-broad! I have no memory of us having that Authorization Domain section. It seems like we should only show one or the other… not both.

Reply:
Also, maybe we should use a similar design to what we have for showing auth domains for group constraint policies? It has a lot more room for the list of groups.

Reply:
It looks like it doesn't appear in policies at all….

```
 {
        "accessLevel": "NO ACCESS",
        "public": false,
        "workspace": {
            "workspaceId": "b4e2fc4d-b8a1-4f04-8758-94ccd62bc4ea",
            "name": "arhPolicyTesting9",
            "state": "Ready",
            "lastModified": "2023-11-15T16:09:24.442Z",
            "attributes": {},
            "cloudPlatform": "Azure",
            "createdBy": "aherbst@broadinstitute.org",
            "namespace": "dsp-dev-testing-20230921",
            "authorizationDomain": [
                {
                    "membersGroupName": "monster-dev"
                },
                {
                    "membersGroupName": "arh-testing-group"
                }
            ]
        }
    },

Reply:
Does it if you include `policies` in the `fields` parameters to the list workspaces API? I don't think that field is returned by default.

Reply:
Yes, it does. Drew said it's expected that it will be returned in both fields.

```
![image](https://github.com/DataBiosphere/terra-ui/assets/484484/6ba9548b-41c0-44e8-ab38-cee63904dbad)